src/pages/en/reference/integrations-reference.md

```diff
@@ -273,11 +274,23 @@ The address, family and port number supplied by the [NodeJS Net module](https://
 
 ```
 
+### `astro:build:generated`
+
+**Previous hook:** [`astro:build:setup`](#astrobuildsetup)
+
+**When:** After a production SSG build has completed.
```

Comment:
We usually prefer the term “static” over SSG in docs as the two SSR + SSG acronyms are a bit too easy to confuse.

```suggestion
**When:** After a static production build has completed.
```

Reply:
Also, would it be more accurate to say something like this?

> After a static production build has finished generating routes and assets.

I’m not familiar with the hook, but sounds like the build has not totally completed at this point?

Reply:
Yep, I like that wording better!  Static builds do a bit of cleanup *before* `astro:build:done` is triggered, the only difference with this new hook is that it runs before the cleanup step